payments-core/src/main/java/com/stripe/android/model/PaymentMethodOptions.kt

```diff
@@ -0,0 +1,57 @@
+package com.stripe.android.model
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.model.StripeModel
+import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.json.JSONObject
+
+/**
+ * The Payment Method Options model.
+ * PaymentIntent PMO: https://stripe.com/docs/api/payment_intents/object#payment_intent_object-payment_method_options
+ * SetupIntent PMO: https://stripe.com/docs/api/setup_intents/object#setup_intent_object-payment_method_options
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Serializable
+@Parcelize
+data class PaymentMethodOptions(
```

Comment:
We don’t want to expose data classes. Any concerns with just exposing everything as a map instead of types?